CMakeLists.txt

```diff
@@ -181,7 +180,14 @@ endif()
 #     products/zerocal.cpp
 #     products/gravitycal.cpp
 #     ... others?
-set(config_dir /etc/barrett/)
+set(BARRETT_ETC_PATH /etc/barrett)
```

Comment:
Have we ever tested the proper operation of local configs stored in ~/.barrett? Have we proven to ourselves that the system will run without the /etc/barrett directory (by deleting /etc/barrett, then testing each example)?